### PR TITLE
pass job location as part of paginated queries if set.

### PR DESCRIPTION
**Pull request by kernel-io**
As per https://github.com/livebook-dev/req_bigquery/issues/33

### CODE REVIEW

**Comment by josevalim** [comment]
It seems we need to bump the Elixir version on CI and mix.exs before moving forward. :)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       matrix:
         include:
           - pair:
-              elixir: '1.12'
+              elixir: '1.13'
```
Comment (josevalim) [inline .github/workflows/ci.yml:17]:
```suggestion
              elixir: '1.15'
```

File: mix.exs [hunk 2]
```diff
@@ -10,7 +10,7 @@ defmodule ReqBigQuery.MixProject do
       version: @version,
       description: @description,
       name: "ReqBigQuery",
-      elixir: "~> 1.12",
+      elixir: "~> 1.13",
```
Comment (josevalim) [inline mix.exs:13]:
```suggestion
      elixir: "~> 1.15",
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by drselump14** [comment]
@josevalim Any plan to release this patch to hex.pm? Without this patch, the rows streams doesn't call the next request on pagination

**Comment by jonatanklosko** [comment]
@drselump14 released in v0.1.5.

**Pull request merged**